scripts/engine/ui/elements/camera.py

```diff
@@ -210,6 +213,19 @@ def update_grid(self):
 
         self._draw_grid(tile_positions)
 
+    def _set_initial_position(self, x: int, y: int):
```

Comment:
Without doing this the camera wouldn't center on the player if the player was in a different position than around (0, 0)